app/main/posts/views/post-view-data.directive.js

```diff
@@ -350,8 +350,7 @@ function PostViewDataController(
     function getNewPosts() {
         var existingFilters = PostFilters.getQueryParams($scope.filters);
         var filterDate = moment(existingFilters.date_before).utc().format();
-
-        if (filterDate >= newPostsAfter) {
+        if (filterDate >= newPostsAfter && existingFilters.order !== 'asc') {
```

Comment:
It might be a little clearer if this was split into its own check at the start of the fn.
```
if (existingFilters.order !== 'asc') {
    $scope.newPostsCount = 0;
}
```